.changeset/metal-suns-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/next": patch
+---
+
+[next][tests] skip some next.js tests
```

Comment:
Shouldn't this be empty if it's just tests?

Reply:
ah yes; if we do skip these, I'll remove that